### DISCUSSION TITLE
VMware Headers

### DISCUSSION THREAD

**Discussion by ghost** [linux-surface/linux-surface#549] [Q&A] [answered]
VMware asks for the Kernel Header location. I haven't been able to find the location and if anyone knows where they are (or how I can get around this message) could you please help me out? 
![vmware-kernel](https://user-images.githubusercontent.com/65696362/131608680-bc2c4821-d3a4-4c54-871d-9980fbbd3b0c.png)

**Comment by qzed** [comment, marked as answer]
Did you install the `kernel-surface-devel` package? I believe that should contain the headers.

**Reply by ghost** [reply]
Installing it seems to have fixed the issue!
